ibm_catalog.json

```diff
@@ -5,6 +5,7 @@
       "name": "deploy-arch-ibm-monitoring-agent",
       "product_kind": "solution",
       "tags": [
+        "Observability",
```

Comment:
Observability is not a valid tag - see `ibmcloud catalog filter options --all` for list of tags. We already have the required tag which is `logging_monitoring`